### ISSUE TITLE
Warlock Fellhunter Quest not available

### ISSUE THREAD

**Issue by doomstyle** [elysium-project/server#3080]
Hello guys,

i tried for several hours now to find out what the problem is.
I can not accept the fellhunter quest with my warlock (lvl 34).

Is this a common problem or individual?
Seems weird as all other warlocks i asked did not have any problems.

**Comment by Nastral** [comment]
I have the exact same issue with my warlock (Elysium PvP).
Also I can not accept the quest for the succubus (Devourer of Souls)

I'm not sure if I abandoned a warlock quest in the past or if they weren't available right away (at least I got the Voidwalker.

I just discussed this issue with an in-game GM who unfortunatly couldn't help me, instead he advised me to report this problem here, which i am doing right now.

**Comment by doomstyle** [comment]
Yep  i tried to delete WDB / WTF, disable / deinstall all add-on and and and
But i still can't do the fellhunter quest....

**Comment by Nastral** [comment]
I solved the problem for me...
Even though I never started the Felhunter quest and have certainly not completed the Succubus questline beyond the first quest, I could start the quests, which appear much later in the actual questline.

So it seems to be a minor problem that the continuity of the class quests seem to be a bit messed up for some people.

**Comment by GuybrushGit** [comment]
Can't reproduce. Most likely you started the Undercity chain and are trying to pick up the Orgrimmar chain, which was closed off by taking the UC one (or viceversa). You need to go back to the first chain of quests you took.
